setup.py

```diff
@@ -11,45 +8,31 @@
     combine_commands,
     get_version,
 )
-
-from setuptools import setup
+import setuptools
 
 
 LONG_DESCRIPTION = 'A Python/ThreeJS bridge utilizing the Jupyter widget infrastructure.'
 
-here = os.path.dirname(os.path.abspath(__file__))
+HERE = Path(__file__).parent.resolve()
 name = 'pythreejs'
-version = get_version(os.path.join(here, name, '_version.py'))
+lab_path = (HERE / name / "labextension")
 
+version = get_version(HERE / name / '_version.py')
 
 cmdclass = create_cmdclass(
     'js',
     data_files_spec=[
-        ('share/jupyter/nbextensions/jupyter-threejs',
-         name + '/static',
-         '*.js'),
-        ('share/jupyter/nbextensions/jupyter-threejs',
-         name + '/static',
-         '*.js.map'),
-        ('share/jupyter/lab/extensions',
-         'js/lab-dist',
-         'jupyter-threejs-*.tgz'),
-        ('share/jupyter/labextensions/jupyter-threejs/',
-         'share/jupyter/labextensions/jupyter-threejs/',
-         '*.*'),
-        ('share/jupyter/labextensions/jupyter-threejs/static',
-         'share/jupyter/labextensions/jupyter-threejs/static/',
-         '*.*'),
-        ('etc/jupyter/nbconfig',
-         'jupyter-config',
-         '**/*.json'),
```

Comment:
why is this removed, i guess we still want that right?

Reply:
yeah, should keep.

Reply:
I put this under test on the [github actions PR](https://github.com/jupyter-widgets/pythreejs/pull/349/files#diff-b803fcb7f17ed9235f1e5cb1fcd2f5d3b2838429d4368ae4c57ce4436577f03fR171)

Reply:
Yes, this was an accidental remove :/ Will fix.

Reply:
Resolved by #8b0a734